docs/source/markdown/podman-image.1.md

```diff
@@ -14,6 +14,7 @@ The image command allows you to manage images
 | Command  | Man Page                                        | Description                                                                 |
 | -------- | ----------------------------------------------- | --------------------------------------------------------------------------- |
 | build    | [podman-build(1)](podman-build.1.md)            | Build a container using a Dockerfile.                                       |
+| diff     | [podman-image-diff(1)](podman-image-diff.1.md)  | Inspect changes on an image's filesystem.                                   |
```

Comment:
need this change in commands-demo.md too

Reply:
I will do this in a different PR.  I want to get this in, so Matt can do a release.